### PR TITLE
Adds basic docs

### PR DESCRIPTION
**Pull request by dkastl**
Changes proposed in this pull request:
- Adds basic docs

### CODE REVIEW

File: doc/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+## Connect SMASH mobile application to Redmine
+
+### Connect Redmine account
+
+In SMASH mobile application top menu go to **Export** or **Import**. Click on the gear icon of **GTT** and you will see the following:
+
+![GTT URL](import_setting.png) ![GTT URL](gtt_url.png)
+
+* **Server URL**: enter your Redmine URL `<account>.lobsta.io`
```
Comment (sanak) [inline doc/getting-started.md]: @dkastl (CC: @nobnisai)  
It's better to use like `http://(your docker-gtt host IP address):3000` .etc.

Comment (dkastl) [reply doc/getting-started.md]: Thanks, just used `example.com`

File: doc/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+* **Server Username**: your Redmine account's `username`
+* **Server Password**: your Redmine account's `password`
+
+### Import Redmine projects
+
+In menu again go to **Import** &#8594; **GTT** &#8594; Select Project &#8594; Click **Import**
+
+![GTT Import](gtt_import.png)
+
+This will import your project's form configuration to mobile storage.
```
Comment (sanak) [inline doc/getting-started.md:42]: @dkastl @nobnisai (CC: @mbasa-georepublic)
To enable downloaded form (`gtt_tags.json`), restarting SMASH app was necessary at last year's SMASH version.
If it is still necessary, maybe better to mention that.

Comment (nobnisai) [reply doc/getting-started.md:42]: @sanak Only the new import is needed. We don't need to restart the app.

Comment (sanak) [reply doc/getting-started.md:42]: @nobnisai Okay, thanks for confirmation!
Then, I resolve this conversation.

**Review by sanak — APPROVED** [review]

**Pull request merged**